testng-core/src/main/java/org/testng/internal/NoOpTestClass.java

```diff
@@ -1,7 +1,9 @@
 package org.testng.internal;
 
+import java.util.List;
 import org.testng.ITestClass;
 import org.testng.ITestNGMethod;
+import org.testng.collections.Lists;
```

Comment:
Avoid `Lists` if possible because there is no more value-add since the diamond operator.